src/lib/openapi/spec/create-api-token-schema.ts

```diff
@@ -1,6 +1,22 @@
 import { FromSchema } from 'json-schema-to-ts';
 import { ApiTokenType } from '../../types/models/api-token';
 
+// TODO: (openapi) this schema isn't entirely correct: `project` and `projects`
+// are mutually exclusive.
+//
+// That is, when creating a token, you can provide either `project` _or_
+// `projects`, but *not* both.
+//
+// We should be able to annotate this using `oneOf` and `allOf`, but making
+// `oneOf` only valid for _either_ `project` _or_ `projects` is tricky.
+//
+// I've opened an issue to get some help (thought it was a bug initially).
+// There's more info available at:
+//
+// https://github.com/ajv-validator/ajv/issues/2096
+//
+// This also applies to apiTokenSchema and potentially other related schemas.
```

Comment:
I'll leave this as is for now. I'll come back and fix this when I go over the entire spec thoroughly.